src/stories/components/UploadPhoto/UploadPhoto.js

```diff
@@ -193,10 +229,14 @@ export const UploadPhoto = () => {
       </div>
       <FileUpload onChange={(e) => uploadFile(e.target)} />
       <div className="upload-photo__submit">
-        <p>Once you are ready. You can upload your photo here.</p>
+        <p>{imageFeedback}</p>
         <Button
           isButton={true}
-          modifierClasses="upload-photo__btn button--small"
+          modifierClasses= {
+            isImageSatisfied ?
+              'upload-photo__btn button--small' :
+              'upload-photo__btn button--small disabled'
```

Comment:
I am a little bit doubtful about the current solution since the `observeStimuliCompletion()` is to listen to the change to the firebase user doc. It will only return 'completed' when called first time. At the 2nd time, it won't catch the change in the document as the change has already happened, so instead, it will continue listening.

So I guess a better solution is that on firebase backend here, I can actually separate the `getSieStimuliFromBucket()` out from the `observeStimuliCompletion()` function. So that frontend won't need to call observe that do the onSnapshot listening 2 times

Reply:
so i did that at first and broke the tests lol and sounds good I will do that per client request. It sounds kinda odd tho IMHO